buildpipeline/perf-pipeline.groovy

```diff
@@ -9,7 +9,7 @@
 def windowsBuild(String arch, String config, String pgo, boolean isBaseline) {
     checkout scm
 
-    String pgoStash = ((pgo == "skiprestoreoptdata") ? "nopgo" : "pgo")
+    String pgoBuildFlag = ((pgo == 'nopgo') ? 'nopgooptimize' : '')
```

Comment:
Just had a thought: we could, and probably should, use the verify PGO support @nategraf has added to make sure we're actually not getting non-PGOized builds when we expect PGOized builds. The default, thus, could be `'enforcepgo'` instead of `''` (on Windows only for now... capability not yet there on Linux to verify).